opengapps-packages.mk

```diff
@@ -1,3 +1,4 @@
+include vendor/google/build/config.mk
```

Comment:
The minus before just means "if we're unable to find the file, ignore it
and continue"

Since we have a file at `vendor/google/config.mk` then that should be
included very early in the build process.

On Tue, 10 Nov 2015, 20:29 mfonville <notifications@github.com> wrote:

> In opengapps-packages.mk
> <https://github.com/opengapps/aosp_build/pull/4#discussion_r44452724>:
>
> > @@ -1,3 +1,4 @@
> > +include vendor/google/build/config.mk
>
> I have literally 0% experience with these .mk files, but the line you
> reference includes a minus-sign before the include, could that have any
> effect?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opengapps/aosp_build/pull/4/files#r44452724>.
>